src/utils/colonyActions.ts

```diff
@@ -345,11 +363,24 @@ export const getSpecificActionValuesCheck = (
         prevColonyTokens ? prevColonyTokens.slice(0).sort() : [],
         currentColonyTokens?.slice(0).sort() || [],
       );
+
+      const removedSafes =
+        (currentColonySafes || []).length < (prevColonySafes || []).length
+          ? (prevColonySafes || []).filter(
+              (safe) =>
+                !(currentColonySafes || []).some(
+                  ({ contractAddress }) =>
+                    contractAddress === safe.contractAddress,
```

Comment:
I think this comparison needs to include a check for the chain as well, since it's possible for two contracts to have the same address but be on different chains, e.g. `0x7E08C7606aCF5887B7dA9914f7c7cD05c2bec8Ab` which is on Ethereum, Polygon and Arbitrum

Reply:
Good point, I was not aware of it